src/site/content/en/progressive-web-apps/app-like-pwas/index.md

```diff
@@ -0,0 +1,745 @@
+  When multitasking, PWAs appear independent from the browser.
+{% endDetails %}
+
+## Quick actions in context menu
+
+The most common app actions, **Search** for new content and **Check for New Episodes**, are available right from the context menu of the app in the Dock. Via the **Options** menu, I can also decide to open the app at login time.
+
+<figure class="w-figure">
+  <img src="./image18.png" alt="Podcasts app icon context menu showing the 'Search' and 'Check for New Episodes' options." class="w-screenshot w-screenshot--filled" width="267">
+  <figcaption class="w-figcaption">Quick actions are immediately available right from the app icon.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  By specifying <a href="https://web.dev/app-shortcuts/">app icon shortcuts</a>
+  in the PWA's web app manifest, you can register quick routes to common tasks that users can reach directly from the app icon.
+  On operating systems like macOS, users can also right-click the app icon and set the app to launch at login time.
+  There is ongoing work on a proposal for <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/tree/master/RunOnLogin">run on login</a>.
+{% endDetails %}
+
+## Act as default app
+
+Other native applications and even websites or emails can integrate with the Podcasts app by leveraging the `podcasts://` URL scheme. If I follow a link like [`podcasts://podcasts.apple.com/podcast/the-css-podcast/id1042283903`](podcasts://podcasts.apple.com/podcast/the-css-podcast/id1042283903) while in the browser, I am brought right into the Podcasts app and can decide to subscribe or listen to the podcast.
+
+<figure class="w-figure">
+  <img src="./image20.png" alt="The Chrome browser showing a confirmation dialog asking the user whether they want to open the Podcasts app.">
+  <figcaption class="w-figcaption">The Podcasts app can be opened right from the browser.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  Handling fully custom URL schemes is not yet possible, but there is ongoing work on a proposal for
+  <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/URLProtocolHandler/explainer.md">URL Protocol Handling</a>
+  for PWAs. Currently, <a href="https://developer.mozilla.org/en-US/docs/Web/API/Navigator/registerProtocolHandler"><code>registerProtocolHandler()</code></a> with a <code>web+</code> scheme prefix is the second best alternative.
+{% endDetails %}
+
+## Native file system integration
+
+You may not immediately think of it, but the Podcasts app naturally integrates with the native file system. When I download a podcast episode, on my laptop it is stored in `~/Library/Group Containers/243LU875E5.groups.com.apple.podcasts/Library/Cache`. Unlike, say `~/Documents`, this directory is of course not meant to be accessed directly by regular users, but it is there.
```

Comment:
For an app like Podcasts is there a benefit to using the Native Filesystem API over IndexedDB? ~/Library isn't designed to be user-visible and is not accessible with the Native Filesystem API.

Reply:
This is more to raise awareness that such an app uses the file system. The prose makes clear it's not meant to be user-visible. The section on [offline content available and media playable](https://deploy-preview-2731--web-dev-staging.netlify.app/app-like-pwas/#offline-content-available-and-media-playable) provides background on the different storage mechanisms and links to an article how to decide for one.

Reply:
My concern is that the text in this section might lead developers down the wrong path if they don't notice the other one. Can you link to that section from this one?

Reply:
Added in https://github.com/GoogleChrome/web.dev/pull/2731/commits/5e22f4654bbe4a98fdd412fc7298fcba5f1e1ef2.